org-gtd-delegate.el

```diff
@@ -38,11 +38,16 @@
 
 ;;;; Customization
 
+(defgroup org-gtd-delegate nil
+  "Customize how tasks within the GTD system are delegated."
+  :group 'org-gtd
+  :package-version '(org-gtd . "3.0.0"))
```

Comment:
Maybe you want to adjust the :package-version value here?

Reply:
I think this didn't exist prior to 3.0.0 but if I'm wrong then yes it should be adjusted.

Reply:
There's no reason for this to exist at the moment. I see you're trying to add the defcustom statement below to it.

If you want to suggest this change, it needs to be in a separate PR so we can talk about it in details, it may trigger further changes.

Reply:
I see. My rationale was that I thought we would all the customizable settings to be visible under the `org-gtd` group in the Customize interface. Is there a way I can remove this change from the PR. Undo the change and push again?

Reply:
Raised as issue #157.